src/pudl/transform/ferc1.py

```diff
@@ -1489,6 +1489,18 @@ def apply_xbrl_calc_fixes(self: Self, tbl_meta: pd.DataFrame) -> pd.DataFrame:
                             "weight": 1.0,
                         },
                     },
+                    {
+                        "calc_component_to_replace": {
+                            "name": "operation_supervision_and_engineering_expense",
+                            "weight": 1.0,
+                            "source_table": [
+                                "plants_steam_ferc1",
+                                "plants_hydro_ferc1",
+                                "plants_pumped_storage_ferc1",
+                            ],
```

Comment:
How does the list of source tables affect the behavior of this replacement operation?

I think it would be more intuitive if we used `source_tables` since this element is a list of tables.

Reply:
the fixer looks for the full calculation component. and these calculation components contain this source table info so it needs to be here for the fixer to actually find the component.

and hard agree on `source_tables` i'll go change that now.